.github/workflows/ci.yml

```diff
@@ -50,12 +55,12 @@ jobs:
         run: |
           sudo apt install -y python3-pathspec
           cd ${{ github.workspace }}/securedrop-workstation-keyring
-          make build-rpm BUILD_CONTAINER=docker BRANCH=${{ github.head_ref || github.ref_name }}
+          make build-rpm BUILD_CONTAINER=docker BRANCH=${{ env.BRANCH_NAME }}
       - name: Upload RPM
         uses: actions/upload-artifact@v4
         id: rpm-upload-step
         with:
-          name: rpm
+          name: rpm-${{ env.BRRANCH_NAME }}
```

Comment:
```suggestion
          name: rpm-${{ env.BRANCH_NAME }}
```

Reply:
ugh, sorry. fixed in [4246ef7](https://github.com/freedomofpress/securedrop-workstation-keyring/commit/4246ef73160bd52c344a519034f780193a00eb78)